static/lang/re-en.json

```
@@ -333,164 +333,164 @@
             },
             "JunkBomb": {
                 "Greater": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*3)[bleed]]] and [[/r (3[splash])[slashing]]]{3 slashing splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 3[bleed]]] and [[/r (3[splash])[slashing]]]{3 slashing splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*3)[bleed]] and @Damage[(3[splash])[slashing]]{3 slashing splash damage}.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[3[bleed]] and @Damage[(3[splash])[slashing]]{3 slashing splash damage}."
                 },
                 "Lesser": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*1)[bleed]]] and [[/r (1[splash])[slashing]]]{1 slashing splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 1[bleed]]] and [[/r (1[splash])[slashing]]]{1 slashing splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*1)[bleed]] and @Damage[(1[splash])[slashing]]{1 slashing splash damage}.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[1[bleed]] and @Damage[(1[splash])[slashing]]{1 slashing splash damage}."
                 },
                 "Major": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*4)[bleed]]] and [[/r (4[splash])[slashing]]]{4 slashing splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 4[bleed]]] and [[/r (4[splash])[slashing]]]{4 slashing splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*4)[bleed]] and @Damage[(4[splash])[slashing]]{4 slashing splash damage}.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[4[bleed]] and @Damage[(4[splash])[slashing]]{4 slashing splash damage}."
                 },
                 "Moderate": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*2)[bleed]]] and [[/r (2[splash])[slashing]]]{2 slashing splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 2[bleed]]] and [[/r (2[splash])[slashing]]]{2 slashing splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[2[bleed]] and @Damage[(2[splash])[slashing]]{2 slashing splash damage}.",
```

```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*2)[bleed]] and @Damage[(2[splash])[slashing]]{2 slashing splash damage}.",
```
Spotted another cheeky typo on a crit 2x entry.

Looks like the last one.